### ISSUE TITLE
Which labels are useful in language tracks?

### ISSUE THREAD

**Issue by kotp** [exercism/discussions#9]
There are two issues in xruby at the moment that are pending some other action.  There is no appropriate label to give for this, to bring (deflect) attention to (from) it.

Also, it may be helpful to have "review this" or some other label to indicate the current status of things.  In progress is the closest one I want to do for the latest clock issue, but it is not quite fitting, due to it is ready for a preliminary review, before I or someone else continues on for the additional changes.

**Comment by kytrinyx** [comment]
How about `status: on hold` or `status: held up` or `status: stuck` or something for the "waiting on some other thing to happen first" label?

I didn't realize that the clock was a preliminary review. I did see that it handled a subset of the new cases, but figured that the other cases would be dealt with in a separate PR. For that I'd want to have `[WIP]` in the subject line of the PR rather than a label, since only committers on the project can add labels, and I think that everyone should be able to submit something for preliminary review.

**Comment by kotp** [comment]
It was just to see if the style was appropriate for what we were testing.  It isn't inappropriate to do it in two PR's.  It is a pretty big change for that file.  Especially since it seems to be outdated by years.

**Comment by kytrinyx** [comment]
I'd like to end up with a label set that we can give each new language track as part of the bootstrapping process in the request-new-language-track repo.

Take a look at the article by `@icecrime`, [A structured approach to labeling](https://icecrime.net/technology/a-structured-approach-to-labeling/) for some inspiration.

What types of categories of labels would be useful? One was mentioned above: `status`. The article mentions two more: area and kind. For us, I think `kind` would make sense (new exercise, exercise fix, continuous integration, end-user documentation, contributing documentation, ...), whereas area doesn't add much value.

But we might have categories that the article doesn't mention (since it's illustrative only). Here's the list of labels for docker/swarm, for inspiration: https://github.com/docker/swarm/labels

Once we have settled on a useful set of labels we can both automate the creation of them for all track repositories, as well as document how to use them in the triaging documentation at https://github.com/exercism/docs/blob/master/maintaining-a-track/triaging-issues.md

**Comment by kytrinyx** [comment]
I think that for now we've more or less got labels that are working for us. I will re-consider a standard set of labels later, but for now I'm going to table this discussion.
